autonomy/cli/analyse.py

```diff
@@ -97,8 +97,23 @@ def check_abci_app_specs(
 ) -> None:
     """Check ABCI app specs."""
 
+    # The command expects 'packages_dir' to be named 'packages', so to make import statements to be compatible with
+    # the standard Python import machinery. The directory can be outside the working directory from which the command
+    # is executed.
+    packages_dir = Path(packages_dir).absolute()
+
+    # add parent directory to Python system path
+    # so to give priority to the import of packages modules
+    sys.path.insert(0, str(packages_dir.parent))
+
+    # make sure the package_dir is named "packages", otherwise
+    # the import of packages.* modules won't work
+    if packages_dir.name != PACKAGES:
+        raise click.ClickException(
+            f"packages directory {packages_dir} is not named '{PACKAGES}'"
+        )
```

Comment:
as I said in the main PR comment, this is a bit restrictive, but it makes things more manageable. It is easy to address by loading `packages_dir/__init__.py` and then `sys.modules["packages"] = <module packages_dir/__init__.py>`